src/main/java/org/openrewrite/java/dropwizard/annotation/micrometer/CodahaleTimedToMicrometerTimed.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.dropwizard.annotation.micrometer;
+
+import org.openrewrite.java.dropwizard.annotation.AnnotationTransformer;
+import org.openrewrite.java.dropwizard.annotation.OneToOneTransformer;
+import org.openrewrite.java.dropwizard.annotation.SerializableFunction;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CodahaleTimedToMicrometerTimed extends AnnotationTransformer {
+
+  @Override
+  protected String getSourceAnnotation() {
+    return "com.codahale.metrics.annotation.Timed";
+  }
+
+  @Override
+  protected String getTargetAnnotation() {
+    return "io.micrometer.core.annotation.Timed";
+  }
+
+  @Override
+  protected Map<String, String> getAttributeMappings() {
+    Map<String, String> mappings = new HashMap<>();
+    mappings.put("name", "value");
+    mappings.put("absolute", "absolute");
+    mappings.put("description", "description");
+    return mappings;
+  }
```

Comment:
Wherever possible we tend to prefer declarative recipes; those building block recipes take care of a lot subtleties behind their minimal API surface, and mean there's less code to maintain in each of the recipe modules. This one in particular looks like it could have been a combination of `org.openrewrite.java.ChangeType` and `org.openrewrite.java.ChangeAnnotationAttributeName`.

Reply:
Implemented just now as an example https://github.com/openrewrite/rewrite-dropwizard/pull/2/commits/898cd9f96f01a07e05cce55b155dd3df78bf21c2
If there's any more that can be similarly replaced that'd be great.